src/main/java/software/amazon/smithy/lsp/ext/LspLog.java

```diff
@@ -90,18 +92,33 @@ private static String currentTime() {
      * @param message object to write, will be converted to String
      */
     public static void println(Object message) {
-        String sanitizedMessage = message != null ? message.toString() : "null";
+        String sanitizedMessage = processMessage(message);
         String timestamped = "[" + currentTime() + "] " + sanitizedMessage;
         try {
             if (fw != null) {
                 fw.append(timestamped + "\n").flush();
             } else {
                 synchronized (buffer) {
-                    buffer.ifPresent(buf -> buf.add(timestamped));
+                    buffer.ifPresent(buf -> buf.add(sanitizedMessage));
                 }
             }
         } catch (Exception e) {
 
         }
     }
+
+    private static String processMessage(Object message) {
```

Comment:
```suggestion
    private static String getStringifiedMessage(Object message) {
```
What do you think about using slightly clearer naming here?

Reply:
Sure no problem